modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -0,0 +1,240 @@
+* *Arrow Size*: Set the size of the arrows on the right and left sides.
+* *Show Dots*: Show dots below the cards. Clicking the dots scrolls through the cards.
+* *Card Height*: Set the card height.
+* *Card Margin*: Set the amount of margin space around each card.
+* *Card Border*: Set the thickness of the card borders.
+* *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
+* *Card Shadow*: Set the shadow beneath each card.
+* *Message If Empty*: Show this message if the carousel has no APIs. Default is "There are no APIs available right now."
+* *Filter Condition (Advanced)*: Filter APIs using package names, field names such as "API name" or "API description", and comparison of strings to those fields.
+// need better info on using Filter condition
+
+== API Console
+
+The interactive API console component shows information about your APIs and their methods, and allows users to test the methods by sending requests.
+
+* *API*: Choose the API to display.
+* *Show Documentation Pages*: Show the documentation pages for the API.
+* *Link Hover Color*: Set the color of links when the cursor hovers over them.
+* *Button Text Color (Optional)*: Set the color of text on buttons.
+* *Button Hover Text Color (Optional)*: Set the color of text on buttons when the cursor hovers over them. Default is the button text color.
+* *Headline Font Family (Optional)*: Set the font family of the API console headline.
+* *Error Color (Optional)*: Set the color of error messages.
+* *Method Label Get Background Color*: Set the background color of the labels of *GET* methods.
+* *Method Label Get Color*: Set the color of the labels of *GET* methods.
+* *Method Label Post Background Color*: Set the background color of the labels of *POST* methods.
+* *Method Label Post Color*: Set the color of the labels of *POST* methods.
+* *Method Label Put Background Color*: Set the background color of the labels of *PUT* methods.
+* *Method Label Put Color*: Set the color of the labels of *PUT* methods.
+* *Method Label Delete Background Color*: Set the background color of the labels of *DELETE* methods.
+* *Method Label Delete Color*: Set the color of the labels of *DELETE* methods.
+* *Method Label Patch Background Color*: Set the background color of the labels of *PATCH* methods.
+* *Method Label Patch Color*: Set the color of the labels of *PATCH* methods.
+* *Method Label Options Background Color*: Set the background color of the labels of *OPTIONS* methods.
+* *Method Label Options Color*: Set the color of the labels of *OPTIONS* methods.
+* *Method Label Head Background Color*: Set the background color of the labels of *HEAD* methods.
+* *Method Label Head Color*: Set the color of the labels of *HEAD* methods.
+
+// Check descriptions: *Headline font family (optional)*
+
+== API Console Documentation
+
+The API console documentation component shows an API's documentation in the API console.
```

Comment:
...shows documentation about APIs.

I wish we could add value by saying something that does not so directly repeat the heading....

Reply:
Updated to "The **API Console Documentation** component shows the documentation from Anypoint Exchange about an API."